instrumentation/github.com/emicklei/go-restful/otelrestful/internal/semconv/httpconv_test.go

```diff
@@ -256,3 +258,18 @@ func TestCurrentHttpClient_MetricAttributes(t *testing.T) {
 		})
 	}
 }
+
+func TestRequestTraceAttrs_HTTPRoute(t *testing.T) {
+	req := httptest.NewRequest("GET", "/high/cardinality/path/abc123", nil)
+	req.Pattern = "/high/cardinality/path/{id}"
+
+	var found bool
+	for _, attr := range (CurrentHTTPServer{}).RequestTraceAttrs("", req) {
+		if attr.Key != semconvNew.HTTPRouteKey {
```

Comment:
Nit: We should use static strings in tests.

Reply:
Thanks, much better.